test/integration/QrochetQuacExt_test.jl

```diff
@@ -0,0 +1,18 @@
+@testset "QrochetQuacExt" begin
+    using Quac
+
+    @testset "QFT_3qubits" begin
```

Comment:
Test set names are for humans to read, so I think this other names are easier.
```suggestion
@testset "Quac" begin
    using Quac

    @testset "QFT" begin
```